elasticdl/python/master/master.py

```diff
@@ -61,5 +102,17 @@ def create_elasticdl_job_service_if_needed(self, args):
             self.elasticdl_job_service = ElasticdlJobService(
                 args, self.task_manager
             )
+            # TODO: Move the initialization of pod manager away from
+            # elasticdl_job_service
+            self.pod_manager = self.elasticdl_job_service.instance_manager
         else:
             self.elasticdl_job_service = None
+
+    def create_master_grpc_servicer(self, args):
```

Comment:
create_master_grpc_servicer -> create_master_grpc_service
because we have created both the GRPC server and the servicer inside this method,